lightspeed/modules/proc_configure-vscode-extension.adoc

```diff
@@ -3,7 +3,7 @@
 [id="configure-vscode-extension_{context}"]
 = Configuring the Ansible VS Code extension
 
-You can configure the Ansible VS Code extension to enable Ansible Lightspeed, and to specify the Ansible Lightspeed portal URL and WCA model ID.
+You can configure the Ansible VS Code extension to enable {LightspeedshortName} and specify it's {LightspeedshortName}portal URL and WCA model ID.
```

Comment:
Existing: "You can configure the Ansible VS Code extension to enable {LightspeedshortName} and specify it's {LightspeedshortName}portal URL and WCA model ID."

Suggested: "You can configure the Ansible VS Code extension to enable {LightspeedshortName} and specify its {LightspeedshortName} portal URL and WCA model ID."

Reply:
Updated.